css/main.css

```diff
@@ -27,7 +27,7 @@ header {
   color: white;
 }
```

Comment:
Please make sure that you are using the class with styles of targeting it directly over the tag name 👍